src/pudl/analysis/timeseries_cleaning.py

```diff
@@ -2149,6 +2159,16 @@ def _score_imputation(
 
         return mape_dict
 
+    @asset_check(
+        asset=imputation_score_asset, name=f"{imputation_score_asset}_asset_check"
+    )
+    def _check_score(mape: float):
+        return AssetCheckResult(
+            passed=mape > settings.simulate_flags_settings.mape_threshold,
```

Comment:
It seems like this check is backwards. We want the measured mape to be less than the threshold don't we?

Reply:
Fixed now.